csfieldguide/chapters/content/en/network-communication-protocols/sections/transport-layer-protocols.md

```diff
@@ -31,12 +31,12 @@ So each page is split up and sent in tiny pieces, then at the other end they are
 
 {panel type="curiosity"}
 
-# Three days?!
+# A full day?!
 
-At 86 terabytes of code, Google's database is one of the largest in the world.
-To write that all down and store in 40ft shipping containers, would create a train as long as the distance from Barcelona to Moscow!
+According to one [2016 estimate](https://www.gearthblog.com/blog/archives/2016/04/big-google-earth-database.html), Google Earth has approximately 3 petabytes (3000000Gb) of past and present image data on our world from above.
```

Comment:
The b in "Gb" needs to be capitalised as a lowercase b means bits and I think you want bytes.